k-distribution/tests/regression-new/issue-1789-rhsOr/haskell/0.test.out

```diff
@@ -0,0 +1,3 @@
+<k>
+  0 ~> .
+</k>
```

Comment:
Sorry, we should instead use `-2` instead of `0`, so that we can test that both sides of the `#Or` are tested in the `split`